src/base-cookie.js

```diff
@@ -23,6 +23,25 @@ const get = (name) => {
   }
 };
 
+const getAll = (name) => {
+  try {
+    const cookieArray = document.cookie.split(';').map((c) => c.trimStart());
+    let values = [];
+    for (let cookie of cookieArray) {
+      while (cookie.charAt(0) === ' ') {
+        cookie = cookie.substring(1);
+      }
```

Comment:
@kelvin-lu any idea why original code in `get` trims spaces off the start of the cookie this way? Is it to maintain backwards compatibility with something? My instinct is to replace it with newer code, but I don't want to break some unspoken contract that's happening here

Reply:
@ajhorst @jooohhn my best guess is that it's cuz `document.cookie` reads (in my chrome browser at least) like:
`cookie_key_one=value_one; cookie_key_two=value_two; ...`
So if we do a split on `;` it would end up producing an extra space on every value after the first.

Reply:
> In the code above allCookies is a string containing a semicolon-separated list of all cookies (i.e. key=value pairs). Note that each key and value may be surrounded by whitespace (space and tab characters): in fact, RFC 6265 mandates a single space after each semicolon, but some user agents may not abide by this.

Probably why it reads a little wonky

Reply:
if the RFC just specifies a single space, then it's fine. I was thinking of a possible case where there are 10 spaces at the beginning of a cookie, and there's this crazy loop taking off exactly one space at a time, but that doesn't sounds likely.